rpm/makefile.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/Makefile b/Makefile
+index 479ad277..2e0ee316 100644
+--- a/Makefile
++++ b/Makefile
+@@ -161,6 +161,24 @@ DEV_VERSION ?= ubuntu_18.04_$(DEV_OPENSSL_VERSION)_gcc9
+ dev:
+ 	@docker run -it --rm --ulimit memlock=-1 -v `pwd`:/home/s2n-dev/s2n $(DEV_IMAGE):$(DEV_VERSION)
+ 
++$(DESTDIR)/usr/lib64:
++	@mkdir -p $(DESTDIR)/usr/lib64
++
++$(DESTDIR)/usr/include:
++	@mkdir -p $(DESTDIR)/usr/include
++
++install: libs $(DESTDIR)/usr/lib64 $(DESTDIR)/usr/include
```

Comment:
i see an example of using `make rpm`. when would one use `install`? if its not used then maybe we get rid of it.

Reply:
It's called by the `%make_install` macro in the spec file.  The paths are AL2 specific, so there is an argument to made these steps could live in the spec file.